### DISCUSSION TITLE
Broken scroll speed on some osu!mania beatmaps with many timing points

### DISCUSSION THREAD

**Discussion by kadoen** [ppy/osu#16400] [Q&A] [answered]
I've found some beatmaps where in osu!lazer the scroll speed is much higher than it should be, even though all their BPM changes are normalized and in stable they work fine, with the same scroll speed as other maps. Specifically these:

 - https://osu.ppy.sh/beatmapsets/1581966#mania/3230598
 - https://osu.ppy.sh/beatmapsets/327318#mania/726470
 - https://osu.ppy.sh/beatmapsets/545484#mania/1166680

What these have in common is a large amount of timing points, but I've tried other maps with many timing points (many more than the #3 example) and they work OK. An example of this would be https://osu.ppy.sh/beatmapsets/1592995#mania/3253520.
This phenomenon has also been tested by 2 other people with the same result.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16406
